compiler/cpp/src/analyzer/Analyzer.cpp

```diff
@@ -1,44 +1,49 @@
-// Copyright (c) Marc Auberer 2021. All rights reserved.
-
 //
-// Created by Marc on 11.05.2021.
+// Created by Marc on 24.05.2021.
```

Comment:
I will look how to disable this comments in CLion, because its not that important to know the date of creation of a file. Will be included in one of the next prs ...

Reply:
Ah, I didn't know that CLion created these comments.